.gitignore

```diff
@@ -23,6 +23,7 @@ make-it/esh
 target-support/
 dist/
 custom.mak
+make-it/custom.mak
```

Comment:
If adding to the .gitignore then also delete custom.mak from the repo and add instructions to the README (preferred)

Otherwise don't include in .gitignore (also fine)